libs/README.md

```diff
@@ -0,0 +1,28 @@
+# Libraries
+
+The code in `libs/` is meant to be reusable libraries that are shared among all
+the boards on the vehicle and the projects that are under development. Examples
+of these would be
+
+* CAN library
+* UART library
+* SPI library
+* GPIO library
+
+Libraries will each have their own subdirectory like so:
+
+```
+libs/
+└── example_library
+    ├── src_arm/         -> Source files for the ARM library
+    ├── src_avr/         -> Source files for the AVR library
+    ├── include/api.h    -> Common public header
```

Comment:
why are we calling it `api.h` rather than `library_name.h'?

Reply:
Just the pattern I've seen. Then we can do something like

```c
#include "libs/can/api.h"
```

Otherwise it would be

```c
#include "libs/can/can.h
```

which feels redundant. But ¯\_(ツ)_/¯

Reply:
yeah i think 

```
#include "libs/can/can.h"
```

is better because then when you open multiple of the header files at once in an editor youd see

`l/c/api.h`, `l/u/api.h`, `l/s/api.h` which is a lot more confusing than `can.h`, `uart.h`, `spi.h`

Reply:
yea I think I agree with wesley here. also, I'd like to be able to include our CAN api with `#include "can.h"` if I can.

Reply:
hm... yeah we can either have `#include "can.h"` or `#include "full/path/to/can.h"`. Not sure yet which will be easier to implement in Bazel.